.circleci/config.yml

```diff
@@ -367,6 +367,8 @@ jobs:
     executor: bionic
     steps:
       - checkout
+      - run: pip2 install flake8==3.7.8
+      - run: pip3 install flake8==3.7.8
```

Comment:
looks unrelated?

Reply:
Yeah, I just noticed we don't pin a version here..  want me to split it out?

Reply:
No, seems fine, I just wanted to make sure it's not unintended.